src/openrct2/actions/ParkEntrancePlaceAction.cpp

```diff
@@ -143,6 +146,11 @@ namespace OpenRCT2::GameActions
                 entranceLoc.y += CoordsDirectionDelta[(_loc.direction + 1) & 0x3].y * 2;
             }
 
+            // Remove any obstructions (e.g. paths)
+            (void)MapCanConstructWithClearAt(
+                { entranceLoc, zLow, zHigh }, MapPlaceParkEntranceClearFunc, { 0b1111, 0 }, flags | GAME_COMMAND_FLAG_APPLY,
```

Comment:
If I’m not mistaken, adding `GAME_COMMAND_FLAG_APPLY` here will cause park entrance ghosts to remove paths and scenery. I haven’t checked yet, though.